workshops/image_editor/README.md

```diff
@@ -0,0 +1,457 @@
+---
+name: 'Image Editor'
+description: 'Make your own image editor'
+author: '@wollygfx'
+img: 'https://cloud-81e0x4zt3.vercel.app/0image.png'
+---
+
+In this workshop you'll learn how to make a very simple image editor using [CSS Filters](https://www.w3.org/TR/filter-effects/) , [jQuery](https://jquery.com) and [Bootstrap](https://getbootstrap.com/docs/4.5/getting-started/introduction/) . Follow along with me and see how easy it is!
```

Comment:
Grammar is a bit off in the last sentence there.

Reply:
Fixed with grammarly.